src/app/tools/actions.ts

```
@@ -0,0 +1,114 @@
+
+  // Apply sorting
+  switch (sort) {
+    case "stars":
+      filteredTools.sort((a, b) => b.githubStars - a.githubStars);
+      break;
+    case "updated":
+      filteredTools.sort((a, b) =>
+        new Date(b.lastUpdate).getTime() - new Date(a.lastUpdate).getTime()
+      );
+      break;
+    case "relevance":
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)